papers/david_dotson/david_dotson.rst

```diff
@@ -397,7 +397,7 @@ Say we would like to get members grouped by both their 'bark' and 'home':
     {('fibrous', 'california'): <Bundle([<Treant: 'sequoia'>])>}
 
 We get only a single member for the pair of keys `('fibrous', 'california')` since 'sequoia' is the only Treant having the 'home' category.
-It is clear that categories are useful by themselves as a means of labeling a Treant to, for instance, denote the types of data that it may contain or the manner in which the data were obtained.
+It is clear that categories are useful by themselves as a means of labeling a Treant to denote the types of data that it may contain or the manner in which the data were obtained.
```

Comment:
Not your fault; this sentence was already too long, and now it feels longer. Need to break it down somehow.